docker/entrypoint-uwsgi.sh

```diff
@@ -31,5 +31,6 @@ exec uwsgi \
   --wsgi dojo.wsgi:application \
   --buffer-size="${DD_UWSGI_BUFFER_SIZE:-8192}" \
   --http 0.0.0.0:8081 --http-to "${DD_UWSGI_ENDPOINT}" \
-  --logformat "${DD_UWSGI_LOGFORMAT:-$DD_UWSGI_LOGFORMAT_DEFAULT}"
+  --logformat "${DD_UWSGI_LOGFORMAT:-$DD_UWSGI_LOGFORMAT_DEFAULT}" \
+  --max-fd "${DD_UWSGI_MAX_FD:-1048576}"
```

Comment:
Is this the default for uswgi? I could not find it online

Reply:
It is what the docker installation defaulted to. Would you rather that we omit the argument instead of forcing a default?

Reply:
I will remove the argument when the `DD_UWSGI_MAX_FD` is not set.